docs/site/migration/tables/lb3-only-commands.html

```diff
@@ -0,0 +1,104 @@
+<head>
+  {% include analytics.html %}
+</head>
+
+<table width="750">
+  <thead>
+    <tr>
+      <th width="200">Commands</th>
+      <th>Description</th>
+      <th>Workaround in LB4</th>
+    </tr>
+  </thead>
+  <tbody>
+
+    <tr>
+      <td>
+        <code>lb bluemix</code>
+      </td>
+      <td> Create a new application destined in Bluemix
+      </td>
+      <td> None
```

Comment:
Does Appsody help to create LB4 app that is ready to deploy to IBM Cloud? 
If not, it might be useful to point it to https://loopback.io/doc/en/lb4/Deploying-to-IBM-Cloud.html.